fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1994,7 +1997,7 @@ private TransactionSynchronizationRegistry getTrxSynchRegistry() throws FHIRPers
                                     ExtractedParameterValue componentParam = parameters.get(0);
                                     // override the component parameter name with the composite parameter name
                                     componentParam.setName(SearchUtil.makeCompositeSubCode(code, componentParam.getName()));
-                                    componentParam.setBase(p.getBase());
+                                    // componentParam.setBase(p.getBase()); TODO not needed?
```

Comment:
do we need to discuss this?

Reply:
This looked to be the only place where getBase/setBase is ever called, so if not needed, it could be completely removed from ExtractedParameterValue, which I can do as part of Issue #2524.

Reply:
Agreed, we should remove this if not needed

Reply:
I removed it